ui/packages/catalog-ui-search/src/main/webapp/plugins/navigator.js

```diff
@@ -0,0 +1 @@
+module.exports = (v) => v
```

Comment:
A comment might be appropriate here to describe the use of this file.

Reply:
All DDF plugins are simply the identity function. All the files in `/plugins` will be identical. Maybe a readme for the directory would be more appropriate.